src/shared-components/alert/index.tsx

```diff
@@ -136,9 +140,7 @@ export const Alert = (alertProps: AlertProps) => {
   );
 };
 
-Alert.Container = ({ children }: { children: React.ReactNode }) => (
```

Comment:
Since this just accepts a node, and a node is overly permissive, I don't think there's any difference between this and using the styled component directly.